src/components/rmrk/Gallery/CollectionItem.vue

```diff
@@ -144,6 +165,11 @@ export default class CollectionItem extends Vue {
   }
 
   get name(): string {
+    if (this.collection.name === undefined) {
+      this.nameLoading = true
+    } else {
+      this.nameLoading = false
+    }
```

Comment:
What will happen if collection doesn't have name?

Reply:
something will not work here, if there is no collection name you will be displaying loading skeleton indefinitely